scripts/db/postgres/dev/create_default_zenysis_admin_user.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+# Create a default admin account for a deployment in the dev database.
+import subprocess
+import sys
+
+from typing import Tuple
+
+from pylib.base.flags import Flags
+from pylib.file.file_utils import FileUtils
+
+
+def guess_zenysis_email(git_email: str) -> str:
+    # Try to figure out what the user's zenysis.com email address is if they have not
+    # set it up in their git config.
+    # NOTE(stephen): Relying on the mostly-well-maintained git mailmap that is used for
+    # internal zenysis pipeline commit stats.
+    mailmap_lines = FileUtils.FileContents(
+        'pipeline/zen/static_data/git-mailmap'
+    ).split('\n')
+
+    email_token = f'<{git_email}>'
+    for line in mailmap_lines:
+        if email_token not in line or '@zenysis.com>' not in line:
+            continue
```

Comment:
This function isn't relevant for Harmony since the hosters won't have zenysis.com addresses. Can you delete this function and rethink the `deduce_user_data` logic? Happy to hop on a call to work through together.

Reply:
We can perhaps hop on a call here, think this was something I just copied from the zen repo to get things working.

However maybe one note, if you see no need for these items you are more than welcome to make commits directly into this PR, or create a PR from this one make all your changes and merge into this one.

Reply:
This isn't really needed, I started copying in stuff whilst testing to get it working. Removed.